Assignment1_q1_alternative_sol.py

```diff
@@ -0,0 +1,16 @@
+from collections import Counter
+
+# Returns true if strings 's1' and 's2' are permutations of the other, false otherwise
```

Comment:
This comments are better suited as docstring comments.
A docstring is a string that is placed as the first statement within a class, method, or function.
Python interprets them in a special way, e.g., the `help()` command will show the docstring for a function.

https://www.python.org/dev/peps/pep-0257/